server/src/main/java/org/apache/druid/query/lookup/LookupUtils.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.lookup;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Maps;
+import org.apache.druid.java.util.emitter.EmittingLogger;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Utility class for lookup related things
+ */
+public class LookupUtils
```

Comment:
*Nit: style choice*

Instead of doing this in a new utils class, would it make sense to add this logic in `LookupReferencesManager` itself?
We could just pass the `LookupState<Object>` to the manager and let it do the rest.

Reply:
@kfaraz, this util function is also used in `LookupListeningResource`, which is why I added to new utils class. I could add another function that just converts a single `LookupState<Object>`, but thought it would be good to have a function that converts a map, as this seems to be done in several places, so the code  can be reused, without re-implementing the logic. Let me know what you think.

Reply:
Yes, makes sense, @zachjsh